dockerbuild/Dockerfile-backend

```diff
@@ -63,8 +64,32 @@ WORKDIR /modeldb/server
 RUN ../scripts/gen_thrift_file.sh java '../thrift/ModelDB.thrift' './src/main/thrift/' && \
     mvn clean compile -Dthrift_version=0.10.0
 
-# This would be better as an ENTRYPOINT with 'mongo' the CMD param, but
-# `exec mvn exec:java ...` leads to a classpath issue that isn't immediately
+
+RUN mvn package -Dthrift_version=${THRIFT_VERSION} -Dmaven.test.skip=true
```

Comment:
This won't work without updates to the POM file (`package` doesn't create a fat jar by default).

Reply:
Added the updates to the POM file